features/car/restrictions.feature

```diff
@@ -506,3 +506,486 @@ Feature: Car - Turn restrictions
             | s    | n  | sj,nj,nj |
             | s    | e  | sj,ej,ej |
 
+    @restriction @compression
+    Scenario: Restriction On Compressed Geometry
+        Given the node map
+            """
+                        i
+                        |
+                    f - e
+                    |   |
+            a - b - c - d
+                    |
+                    g
+                    |
+                    h
+            """
+
+        And the ways
+            | nodes |
+            | abc   |
+            | cde   |
+            | efc   |
+            | cgh   |
+            | ei    |
+
+        And the relations
+            | type        | way:from | node:via | way:to | restriction   |
+            | restriction | abc      | c        | cgh    | no_right_turn |
```

Comment:
We should just ignore them. I would consider these a mapping error to begin with and not worry about them.
Internally

`a - b - c` and `ab` via `b` to `bc` should turn out as `ac` via `b` to `ac` after compression and be ignored as inconsistent.